packages/create-sitecore-jss/src/templates/angular-xmcloud/README.md

```diff
@@ -0,0 +1,58 @@
+
+- `XMCloud Angular`: Adds support for the Sitecore Context data, which simplifies connecting the application to XM Cloud and configuring the integration of multiple composable Sitecore products. Also includes example components for SXA projects.
+
+- `XMCloud Proxy`: Adds integration with XMCloud for the JSS SPA applications. Considered experimental.
+
+## Environment Variables
+
+The following environment variables can be set to configure the angular app. You can use the `.env` file located in the root of the app or set these directly in the environment (for example, in containers).
+
+| Parameter                              | Description                                                                                                                                |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ |
+| `PROXY_HOST`                        | Your XM Cloud Proxy hostname is needed to build the app and execute the client-side requests against the proxy server.                                                                                                                  |
+| `PROXY_BUILD_PATH`                              | Your XM Cloud Proxy server path is needed to build the app. The build output will be copied to the proxy server path.
+| `SITECORE_EDGE_CONTEXT_ID`                              | The Context ID, which covers many system configurations, required for connecting to the XM Cloud back end. This is an XM Cloud system environment variable. When the application runs on the XM Cloud rendering host, this value is always set to the preview Context ID.                   |
+| `SITECORE_API_KEY`                              | The Context ID, which covers many system configurations, required for connecting to the XM Cloud back end. This is an XM Cloud system environment variable. When the application runs on the XM Cloud rendering host, this value is always set to the preview Context ID.                   |
+| `SITECORE_API_HOST`                              | The API hostname, needed to build the application. When the application runs on the XM Cloud rendering host, this value is always set to your content management instance, and it will override any value set in your .env file. For example, https://<xmc_cm_host>.sitecorecloud.io.                   |
+| `GRAPH_QL_ENDPOINT`                              | The GraphQL endpoint. For Experience Edge, the value is https://edge.sitecorecloud.io/api/graphql/v1. For a preview endpoint (a CM instance on XM Cloud or a local one), the value is <xmc_cm_host>/sitecore/api/graph/edge. When the application runs on the XM Cloud rendering host, this value is always set to the preview endpoint for your CM instance <xmc_cm_host>/sitecore/api/graph/edge , and it will override any value set in your .env file.    |
```

Comment:
Do we need to provide this description? api host + api key setup is sufficient for local/dev/container setup and describing graphql endpoint too may be confusing

Reply:
I think since we provide the env variable out of the box we need to explain its purpose and I think the explanation clarifies the difference rather than being confusing.